cli/config/configuration-qemu.toml.in

```diff
@@ -89,6 +89,9 @@ default_memory = @DEFMEMSZ@
 # Default 0
 #memory_offset = 0
 
+# Specify whether cpu/memory used by guest is counted in container resource quota.
+#overhead_in_quota = false
```

Comment:
@jodh-intel  we should really unify this into one config and customization ? Or something like that

Reply:
Hi @jcvenegas - slightly confused - are you suggesting we have separate options for CPU and memory?